android/libraries/rib-base/src/main/java/com/badoo/ribs/core/routing/action/RoutingAction.kt

```diff
@@ -5,7 +5,10 @@ import com.badoo.ribs.core.Node
 
 interface RoutingAction<V : RibView> {
 
-    fun ribFactories() : List<() -> Node<*>> =
+    val allowAttachView: Boolean
```

Comment:
Need a comment of documentation for clarifying.

Reply:
Will do, thanks.

Reply:
Removed global flag (wouldn't work with composite routing actions if there's a mix of true and false), added enum for attach mode in Node (allows granular control, each individual child can be marked how to attach) with proper comments.